lisa/messages.py

```diff
@@ -137,3 +131,71 @@ class NetworkPPSPerformanceMessage(PerfMessage):
     rx_tx_pps_minimum: Decimal = Decimal(0)
     rx_tx_pps_average: Decimal = Decimal(0)
     rx_tx_pps_maximum: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkNtttcpTCPPerformanceMessage(PerfMessage):
+    connections_num: int = 0
+    throughput_in_gbps: Decimal = Decimal(0)
+    latency_us: Decimal = Decimal(0)
+    buffer_size: Decimal = Decimal(0)
+    tx_packets: Decimal = Decimal(0)
+    rx_packets: Decimal = Decimal(0)
+    pkts_interrupts: Decimal = Decimal(0)
+    number_of_receivers: int = 1
+    number_of_senders: int = 1
+    sender_cycles_per_byte: Decimal = Decimal(0)
+    connections_created_time: int = 0
+    retrans_segments: int = 0
+    receiver_cycles_rer_byte: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkNtttcpUDPPerformanceMessage(PerfMessage):
+    connections_num: int = 0
+    number_of_receivers: int = 1
+    number_of_senders: int = 1
+    connections_created_time: int = 0
+    receiver_cycles_rer_byte: Decimal = Decimal(0)
+    send_buffer_size: Decimal = Decimal(0)
+    tx_throughput_in_gbps: Decimal = Decimal(0)
+    rx_throughput_in_gbps: Decimal = Decimal(0)
+    data_loss: Decimal = Decimal(0)
+    packet_size_kbytes: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkIperfSingleTCPPerformanceMessage(PerfMessage):
+    buffer_size_bytes: Decimal = Decimal(0)
+    tx_throughput_in_gbps: Decimal = Decimal(0)
+    rx_throughput_in_gbps: Decimal = Decimal(0)
+    retransmitted_segments: Decimal = Decimal(0)
+    congestion_windowsize_kb: Decimal = Decimal(0)
+
+
+@dataclass
+class NetworkIperfUDPPerformanceMessage(NetworkNtttcpUDPPerformanceMessage):
+    pass
+
+
+def create_message(
+    message_type: Type[T],
+    node: "Node",
+    information: Dict[str, str],
```

Comment:
Because the information is always from the environment, so we can pass in the environment every time.

Reply:
updated to use environment